Formula/etcher-cli.rb

```diff
@@ -0,0 +1,32 @@
+require "language/node"
+
+class EtcherCli < Formula
+  desc "Flash OS images to SD cards & USB drives, safely and easily"
+  homepage "https://etcher.io/"
+  url "https://github.com/resin-io/etcher/archive/v1.4.4.tar.gz"
+  sha256 "02082bc1caac746e1cdcd95c2892c9b41ff8d45a672b52f8467548cad4850f5d"
+
+  depends_on "node@6" => :build # must be abi compatiable to the node version used with pkg
+  depends_on "python" => :build
+
+  def install
+    ENV["NODE_ENV"] = "production"
+    system "npm", "install", *Language::Node.local_npm_install_args
+    inreplace buildpath/"node_modules/lzma-native/index.js",
+              "var native = require(binding_path);",
+              "var native = require(path.join(__dirname, 'binding', 'lzma_native.node'));"
```

Comment:
I think this patch is only required for the `electron` build and not for the `pkg` build. Could you try to remove it and test if flashing a image is working with a build without the`inreplace`.